sites/fast-website/src/app/components/design-system-provider/website-design-system-provider.ts

```diff
@@ -0,0 +1,23 @@
+import { ColorRGBA64, parseColor } from "@microsoft/fast-colors";
+import {
+    designSystemProperty,
+    designSystemProvider,
+    FASTDesignSystemProvider,
+} from "@microsoft/fast-components";
+import { createColorPalette } from "@microsoft/fast-components-styles-msft";
+import { observable } from "@microsoft/fast-element";
+
+@designSystemProvider("website-design-system-provider")
+export class WebsiteDesignSystemProvider extends FASTDesignSystemProvider {
+    @observable
+    @designSystemProperty({
+        cssCustomProperty: false,
+        default: createColorPalette(parseColor("#F33378") as ColorRGBA64),
+    })
+    public accentPalette: string[];
+
+    accentBaseColorChanged(): void {
+        const parsedColor = parseColor(this.accentBaseColor);
```

Comment:
Can you make this handle the null case more gracefully? If the color can't be parsed then maybe it is reset back to the previous color?

Then you can diff the previous and next values, and only create the accent palette when the two are different and current is a valid color

Reply:
Which null cases do you mean? When the attribute is present but it's empty, when the attribute isn't present, or when the attribute is set to an invalid color value?